client/src/pages/Balance.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useContext } from "react";
+import axios from "axios";
+import {
+  makeStyles,
+  Button,
+  Grid,
+  Typography,
+  TextField,
+  FormHelperText,
+} from "@material-ui/core";
+import MainContainer from "components/MainContainer";
+import GridTemplateContainer from "components/GridTemplateContainer";
+
+import UserContext from "context/UserContext";
+
+import AddBoxIcon from '@material-ui/icons/AddBox';
+import IndeterminateCheckBoxIcon from '@material-ui/icons/IndeterminateCheckBox';
+
+// TO DO: figure out where to move useStyles to avoid duplicate code
```

Comment:
You can actually see what Frank does in another PR with the submitButton

Reply:
Fixed